### ISSUE TITLE
Editor view chops off text early

### ISSUE THREAD

**Issue by rfinnie** [retext-project/retext#475]
Reference markdown file: https://tangentsoft.com/pidp8i/raw/README.md?name=f50fdb25dcc4d477d7be3e558e5b40080cd5f71567c3a64a31126e76945e3138

When loading this file in retext, the editor window ends at the following lines, which is far from the bottom of the file:
```
1.  To start the simulator running in the background:

        $ pidp8i start
```

When I switch to Preview, it renders the entire file.  Interestingly, when I switch back out of Preview, the editor then shows the entire file.

**Comment by xgouchet** [comment]
I tried reproducing this issue, but didn't see it on the latest commit on master, on a XUbuntu 18.04. 

@rfinnie what version of Retext and what OS are you using it on ?

**Comment by mitya57** [comment]
I also cannot reproduce this. Maybe this is a variant of #345 which was fixed in ReText 7.0.2?
